rcl/include/rcl/timer.h

```diff
@@ -590,6 +599,33 @@ RCL_WARN_UNUSED
 rcl_guard_condition_t *
 rcl_timer_get_guard_condition(const rcl_timer_t * timer);
 
+/// Set the on reset callback function for the timer.
+/**
+ * This API sets the callback function to be called whenever the
+ * timer is reset.
+ *
```

Comment:
i think it would be better to add doc that callback in argument will be issue via this API if timer has been already reset.

Reply:
Done in 9f5c847